lib/services/matchers.go

```diff
@@ -353,10 +353,13 @@ var SupportedAzureMatchers = []string{
 const (
 	// GCPMatcherKubernetes is the GCP matcher type for GCP kubernetes.
 	GCPMatcherKubernetes = "gke"
+	// GCPMatcherVM is the GCP matcher for GCP VMs.
+	GCPMatcherVM = "vm"
```

Comment:
vm is quite broad term. What would you say about replacing  this matcher name with `gce` - Google Compute Engine ?